### ISSUE TITLE
The MetaQA dataset

### ISSUE THREAD

**Issue by LinxiCai** [malllabiisc/EmbedKGQA#125]
when I read the train.txt, valid.txt and test.txt in data/MetaQA, I found the triples in test.txt are included in train.txt, could you explain why should this happen?

**Comment by apoorvumang** [comment]
Hi LinxiCai, thanks for your interest.

We studied MetaQA for the QA task, not KG completion task. We want to pretrain on the whole KG (or 50% KG depending on setting) and then finetune for QA. test.txt and valid.txt triples exist just for compatibility with KGE implementations, which require separate validation and test triples. So we simply copied triples from train.txt to test.txt to maintain compatibility.

**Comment by LinxiCai** [comment]
OK,thanks a lot !! I understand. By the way, I had another question, when I train embedding for metaQA triples, if I don't use dropout or batch_normalization, will there be overfitting? or can you share your training arguments when you get your MetaQA embedding?
